cluster/wnode/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  bootnode:
+    image: ethereum/client-go:alltools-latest
+    entrypoint: bootnode
+    command: -addr=:30303 -nodekey /static/keys/bootnode.key
+    volumes:
+      - ./../../static/keys:/static/keys:ro
+    networks:
+      cluster:
+        ipv4_address: 172.16.238.10
+
+  wnode:
+    image: status-go:latest
```

Comment:
It might be nice to add `depends_on` here to make sure the startup order is correct.